### PR TITLE
Updated sendgrid documentation for issue #2452 related to dapr/compon…

### PR DESCRIPTION
**Pull request by arahja**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Updated SendGrid Dapr Component Documentation related to dapr/components-contrib#1749

## Issue reference

Please reference the issue this PR will close: #2452

### CODE REVIEW

**Comment by msfussell** [comment]
@arahja - Could you also update the metadata spec at the start of the document to add the following fields

emailFromName 
emailToName 
emailCc 
emailBcc 

Add them into this below as an example.

 metadata:
  - name: emailFrom
    value: "testapp@dapr.io" # optional
  - name: emailTo
    value: "dave@dapr.io" # optional
  - name: subject
    value: "Hello!" # optional
  - name: apiKey
    value: "YOUR_API_KEY"

**Comment by arahja** [comment]
I made the requested changes.

**Comment by msfussell** [comment]
> I made the requested changes.

Excellent. One more question. Can the email addresses be a list of addresses? For example can I have 2 or more emails on the To, CC and BCC fields?  I was asked this questions yesterday and the docs are not clear, if these can be a list. If so, can you update the text and provide a list example. If not, just say that one one email address is allowed for these.

**Comment by arahja** [comment]
I will make the change in the documentation to state that only one email address can be used. I have been contemplating making a change to allow multiple email addresses. However, that is a breaking change so I haven't attempted that yet. I need figure out the [v2 process](https://github.com/dapr/components-contrib/blob/master/docs/developing-component.md#version-2-and-beyond-of-a-component).

File: daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md [hunk 1]
```diff
@@ -42,7 +50,9 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 |--------------------|:--------:|------------|-----|---------|
 | apiKey | Y | Output | SendGrid API key, this should be considered a secret value | `"apikey"` |
 | emailFrom | N | Output | If set this specifies the 'from' email address of the email message. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md]:
```suggestion
| emailFrom | N | Output | If set this specifies the 'from' email address of the email message. Only a single email address is allowed. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md [hunk 2]
```diff
@@ -42,7 +50,9 @@ The above example uses secrets as plain strings. It is recommended to use a secr
...
+| emailFromName | N | Output | If set this specifies the 'from' name of the email message. Optional field, see [below](#example-request-payload) | `"me"` |
 | emailTo | N | Output | If set this specifies the 'to' email address of the email message. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md]:
```suggestion
| emailTo | N | Output | If set this specifies the 'to' email address of the email message. Only a single email address is allowed. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md [hunk 3]
```diff
@@ -42,7 +50,9 @@ The above example uses secrets as plain strings. It is recommended to use a secr
...
+| emailToName | N | Output | If set this specifies the 'to' name of the email message. Optional field, see [below](#example-request-payload) | `"me"` |
 | emailCc | N | Output | If set this specifies the 'cc' email address of the email message. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md]:
```suggestion
| emailCc | N | Output | If set this specifies the 'cc' email address of the email message. Only a single email address is allowed. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md [hunk 4]
```diff
@@ -42,7 +50,9 @@ The above example uses secrets as plain strings. It is recommended to use a secr
...
 | emailBcc | N | Output | If set this specifies the 'bcc' email address of the email message. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/sendgrid.md]:
```suggestion
| emailBcc | N | Output | If set this specifies the 'bcc' email address of the email message. Only a single email address is allowed. Optional field, see [below](#example-request-payload) | `"me@example.com"` |
```

**Comment by msfussell** [comment]
> I will make the change in the documentation to state that only one email address can be used. I have been contemplating making a change to allow multiple email addresses. However, that is a breaking change so I haven't attempted that yet. I need figure out the [v2 process](https://github.com/dapr/components-contrib/blob/master/docs/developing-component.md#version-2-and-beyond-of-a-component).

I suggested a simple sentence to add to the description. PTAL.

**Comment by arahja** [comment]
Yes, sorry. Thanks for the help.

**Review by msfussell — APPROVED** [review]
LGTM

**Comment by msfussell** [comment]
@arahja - Thanks for the contribution. Really appreciate this.

**Pull request merged**